### PR TITLE
Release notes for Opencast 8.0

### PR DESCRIPTION
**Pull request by rrolf**

### CODE REVIEW

File: docs/guides/admin/docs/releasenotes.md [hunk 1]
```diff
@@ -4,15 +4,65 @@ Opencast 8: Release Notes
...
+  - Hourly scale for data exports
+  - Visualization of statistics in the Admin UI
+- Limit accepted file types when uploading assets
+- Support for exclusion pattern for URL signing
+- Add option to configure state mappings for workflows
+- Assembly configuration
+- Multi-tenancy support for workflows
+- Role support for workflows
+- Video-crop service
+- Paella Player 6.2.2
```
Comment (karendolan) [inline docs/guides/admin/docs/releasenotes.md]: The current Paella player is 6.2.4
```
commit 9e9af9993a01a778c16b2707d1b310095ac52c80
Author: Miguel Escriva <miesgre@gmail.com>
Date:   Tue Nov 5 14:26:23 2019 +0100

    Update to paella 6.2.4
```

File: docs/guides/admin/docs/releasenotes.md [hunk 2]
```diff
@@ -4,15 +4,65 @@ Opencast 8: Release Notes
...
+- Multiple audio tracks support on paella
```
Comment (karendolan) [inline docs/guides/admin/docs/releasenotes.md]: Maybe capitalize paella ~ Paella Player.

File: docs/guides/admin/docs/releasenotes.md [hunk 3]
```diff
@@ -4,15 +4,65 @@ Opencast 8: Release Notes
...
+- Media Module configuration now in the organization configuration file
+- Sensible names for hosts instead of URLs
+- Improved icons and wording in video editor
+- Improved delete-event submit button
+- Extended the ingest-download-woh
+- Tag elements retrieved from asset manager
+- Improve navigation in video editor when zoom is active
+- Switch to compatible file type filter definitions
+- Improved setting values from Dublin Core catalog
+- Don't start opencast on a used port
```
Comment (karendolan) [inline docs/guides/admin/docs/releasenotes.md]: Maybe capitalize "Opencast"

File: docs/guides/admin/docs/releasenotes.md [hunk 4]
```diff
@@ -27,17 +77,19 @@ API changes
...
 
-Finally, Opencast 7.4 also brings an update to the CAS security example that was out of date.
 
-Release Schedule
+Release managers
 ----------------
 
-TBD
+- Karen Dolan (Harvard DCE)
+- Rüdiger Rolf (University of Osnabrueck)
```
Comment (karendolan) [inline docs/guides/admin/docs/releasenotes.md]: Maybe "(Harvard University DCE)"

File: docs/guides/admin/docs/releasenotes.md [hunk 5]
```diff
@@ -4,15 +4,65 @@ Opencast 8: Release Notes
 Features
 --------
 
-- The `adminworker` distribution is no longer available.
+- Improved integration of external statistics providers (InfluxDB)
+  - Hourly scale for data exports
+  - Visualization of statistics in the Admin UI
```
Comment (karendolan) [inline docs/guides/admin/docs/releasenotes.md]: The Buildbot is complaining about more spaces on this indent: "Unordered list indentation [Expected: 4; Actual: 2]"

**Comment by karendolan** [comment]
HI @rrolf, a made a couple small notes. The Paella version and extra indents seem to be the most important.

File: docs/guides/admin/docs/releasenotes.md (hunk 4 above)
Comment (lkiesow) [reply docs/guides/admin/docs/releasenotes.md]: And `Osnabrück University` ;-)

File: docs/guides/admin/docs/releasenotes.md (hunk 5 above)
Comment (gregorydlogan) [reply docs/guides/admin/docs/releasenotes.md]: So is Travis ;)

**Pull request merged**